### ISSUE TITLE
[Q] is it possible to have 2 networks connected remotely?

### ISSUE THREAD

**Issue by alexbarnsley** [PrinceOfAmber/Storage-Network#448]
Minecraft Version: 1.18.2

Forge Version: 40.1.0

Mod Version: 1.6.1

Single Player or Server: Server

Describe problem (what you were doing; what happened; what should have happened):

I would like to connect 2 different areas to the same network. Is this possible to do, or could this be a potential feature for the future? 

Scenario:
I have 1 storage network at my main base, then a second base which runs automated farms (e.g. a cobblestone farm). I would like to link the cobblestone storage at the automated farm with the main storage network without having to run a very long set of network cables. Thanks

**Comment by Lothrazar** [comment]
This is an upgrade ive been wanting to do for a while, finding a way to make that work. have two networks not connected by cables but some wireless system. maybe with built in chunk loading

Maybe something like a special "cable" , but instead of connecting to a chest it somehow connects to one normal network and one remote network.  meaning this wireless-cable needs to know the x/y/z/dimension of the Main master tile of the other network.  so player needs a way to connect the data and then it would work like an import/export cable? 
also how would it show up in a GUI on the main network. would there be two tabs? Would items be mixed together or is that possible.  or maybe the sub-network only works as imports and not in the screen.  

Its a cool upgrade but lots of details to think about

**Comment by alexbarnsley** [comment]
Apologies for the delay, it's great that this is something on the cards! 

The way envisioned it would work is that you would have the master node for the network as normal, you would branch a cable off of the main network into a new "wireless" node which can be given some sort of ID or name. Then at the place you want to attach to the network, you would create a second "wireless" node which could connect to the first via ID/name, or some sort of magic in the background (e.g. list all wireless nodes it can connect to). Wireless node B could then be branched off into each storage container with the same cables as it does currently in the main network

Basically treat the new node the same as a wireless bridge would work in real-life wireless networks

Hopefully that makes sense
